### PR TITLE
[11.0][PORT] porting pos_longpolling from 10.0 to 11.0

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_longpolling/__manifest__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# -*- coding: utf-8 -*-
+
```
Comment (yelizariev) [inline pos_longpolling/__manifest__.py]: Fix that

**Review by yelizariev — DISMISSED** [review]
Seems fine

**Review by yelizariev — COMMENTED** [review]
seems ok

**Review by Railia — APPROVED** [review]

**Comment by ilmir-k** [comment]
@GabbasovDinar I'm not able to use "Deactivate empty order" field in pos.config

**Comment by GabbasovDinar** [comment]
@ilmir-k What was the error?

**Comment by ilmir-k** [comment]
@GabbasovDinar It's ok and no need any action. I found out that it is editable when I uncheck `Replace empty order` (i.e. `multi_session_replace_empty_order=False`).

**Comment by ilmir-k** [comment]
@yelizariev it can be merged

**Review by ilmir-k — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

**Comment by yelizariev** [comment]
@ilmir-k merge yourself

**Pull request merged**